pandas/tests/test_base.py

```diff
@@ -1175,3 +1176,136 @@ def test_iter_box(self):
             assert isinstance(res, pd.Period)
             assert res.freq == 'M'
             assert res == exp
+
+
+@pytest.mark.parametrize('arr, expected', [
```

Comment:
this is not the right location for these. they need to go in appropriate test locations.